actor/builtin/miner/miner.go

```diff
@@ -145,23 +151,23 @@ type State struct {
 	SectorSize *types.BytesAmount
 }
 
-// NewActor returns a new miner actor
-func NewActor() *actor.Actor {
-	return actor.NewActor(types.MinerActorCodeCid, types.NewZeroAttoFIL())
+// NewActor returns a new miner actor with the provided balance.
+func NewActor(balance *types.AttoFIL) *actor.Actor {
```

Comment:
This is a little sketchy. This only gets used in tests an genesis, but it suggests you can create a miner actor in a state transition with a non-zero balance. This would be really bad, since it would be generating FIL from nowhere. For clarity's sake, I'd rather you not add balance to the constructor and set it manually after the actor is constructed.

Reply:
> For clarity's sake, I'd rather you not add balance to the constructor and set it manually after the actor is constructed.

OK. This change is easy to roll back. I introduced it to [be consistent](https://github.com/filecoin-project/go-filecoin/blob/master/actor/builtin/account/account.go#L23) with the account actor-creation constructor. Do you think that the same logic applies to the account actor?